mlir/include/mlir/Dialect/Bufferization/IR/BufferizationOps.td

```diff
@@ -493,6 +494,7 @@ def Bufferization_ToTensorOp : Bufferization_Op<"to_tensor", [
 // ToMemrefOp
 //===----------------------------------------------------------------------===//
 
+// TODO: rename to "to_buffer"
```

Comment:
side-note: I would prefer to do it in a separate patch (either before or after this one) since renaming would be (almost?) NFC